shell/README.md

```diff
@@ -31,3 +31,10 @@ the following snippet
 #### Oh my Zsh
 
 If you are using the popular [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh) framework to manage your zsh plugins, move the file `exercism_completion.zsh` into `~/.oh-my-zsh/custom`.
+
+### Fish
+
+Completions must go in the user defined `$fish_complete_path`. By default, this is `~/.config/fish/completions`
+
```

Comment:
We could do that, but the final file has to have the same name as the executable. I figured it would be easier to have it named that way from the start, rather than requiring users to rename it themselves. I foresee at least a few people dragging and dropping rather than using the command line.